homeassistant/components/miele/sensor.py

```diff
@@ -249,3 +338,15 @@ def available(self) -> bool:
         """Return the availability of the entity."""
         # This sensor should always be available
         return True
+
+
+class MieleDurationSensor(MieleSensor):
+    """Representation of the duration sensor."""
+
+    @property
+    def native_value(self) -> StateType:
+        """Return as minutes."""
+        value_list = cast(list[int], self.entity_description.value_fn(self.device))
+        if len(value_list) == 0:
+            return None
+        return value_list[0] * 60 + value_list[1]
```

Comment:
```suggestion
        return value_list[0] * 60 + value_list[1] if value_list else None
```
but I think we could just make this a helper function which we call in all the duration lambdas saving us:
- the extra class
- casting the "normal" sensor